packages/math/__tests__/apr.test.ts

```diff
@@ -78,18 +86,21 @@ describe("Test APR calculations", () => {
         (item) => item.pool_id === pool.id
       )!.volume_7d;
 
-      const aprs = calcPoolAprs({
-        activeGauges,
-        lockupDurations,
-        pool,
-        prices,
-        superfluidPools,
-        swapFee: pool.poolParams!.swapFee,
-        volume7d,
-        aprSuperfluid: 5.28,
-        includeNonPerpetual: opts.includeNonPerpetual,
-        lockup: opts.lockup,
-      });
+      const aprs = calcPoolAprs(
+        osmosisAssets,
```

Comment:
can I ask, why not include the property in the object below, maybe `assets`?

Reply:
oh it is my oversight.
it will be better to include an assets property in the parameter object.